components/support/ktx/src/main/java/mozilla/components/support/ktx/android/view/Window.kt

```diff
@@ -17,7 +17,7 @@ import mozilla.components.support.utils.ColorUtils.isDark
  */
 fun Window.setStatusBarTheme(@ColorInt color: Int) {
```

Comment:
The argument name could be renamed to `toolbarColor` so it should be clear that the decision to have a light stats bar or not is made based on the custom tab toolbar color.

Reply:
Done